resources/views/teams/show.blade.php

```diff
@@ -124,14 +124,28 @@ class="team-action-button team-action-button--join js-login-required--click"
                             </div>
                             @if (present($team->url))
                                 <div class="team-info-entry">
-                                    <div class="team-info-entry__title">{{ osu_trans('teams.show.info.website') }}</div>
-                                    <div class="team-info-entry__value">
-                                        <span class="u-ellipsis-overflow">
-                                            <a href="{{ $team->url }}">{{ $team->url }}</a>
-                                        </span>
+                                    <div class="team-info-entry__title">{{ osu_trans('teams.edit.settings.url') }}</div>
```

Comment:
If we're sharing keys between edit/show/whatever, should they even be in that namespace?